src/org/mozilla/javascript/Hashtable.java

```diff
@@ -1,30 +1,21 @@
 package org.mozilla.javascript;
 
 import java.io.Serializable;
-/* -*- Mode: java; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 4 -*-
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
```

Comment:
Why did spotless remove this comment block?

Reply:
that was my fault, sorry
will have a look